app/main.py

```diff
@@ -1,13 +1,104 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors":
+            [
+                {
+                    "line": errors[0]["line_number"],
+                    "column": errors[0]["column_number"],
+                    "message": errors[0]["text"],
+                    "name": errors[0]["code"],
+                    "source": "flake8"
+                },
+                {
```

Comment:
Don't repeat yourself. Use `format_linter_error` function to format a single error